lib/bolt/outputter/human.rb

```diff
@@ -78,6 +78,8 @@ def handle_event(event)
           @disable_depth += 1
         when :message
           print_message(event[:message])
+        when :verbose
```

Comment:
This also needs a handler in the JSON outputter `https://github.com/puppetlabs/bolt/blob/main/lib/bolt/outputter/json.rb#L24`

Reply:
We'll also want to update the CLI so `:verbose` events are subscribed to when using the JSON outputter:
https://github.com/puppetlabs/bolt/blob/main/lib/bolt/cli.rb#L791